gensim/corpora/textcorpus.py

```diff
@@ -112,7 +114,10 @@ class TextCorpus(interfaces.CorpusABC):
     6.  remove stopwords; see `gensim.parsing.preprocessing` for the list of stopwords
 
     """
-    def __init__(self, input=None, dictionary=None, metadata=False, character_filters=None, tokenizer=None, token_filters=None):
+    def __init__(
```

Comment:
Please use vertical indent (**only** for method/function definition, in all other cases - hanging indent).